athena-elasticsearch/README.md

```diff
@@ -66,11 +66,17 @@ endpoint.
     `=` are used by this connector as separators for the domain-endpoint pairs. Therefor, they 
     should **NOT** be used anywhere inside the stored secret.
 
-5. **spill_bucket** - When the data returned by your Lambda function exceeds Lambda’s limits, 
+4. **query_timeout_cluster** - timeout period (in seconds) for Cluster-Health queries used in the
+generation of parallel scans.
+
+5. **query_timeout_search** - timeout period (in seconds) for Search queries used in the retrieval
+of documents from an index.
```

Comment:
Are these optional? Are these optional? if not, they should be and we should offer sane default values. Also, can you explain the search timeout a bit more? what specific activity does this control? Is this the maximum duration of a search? e.g. any search taking longer than this will fail due to timeout? or is this about connection health and this even a 10 second timeout can be OK fo queries taking 100 minutes to complete.

Reply:
Normally, if there's a communication/health issue, an exception will be thrown. However, if the query takes a significantly long time due to limited resources and/or high churn rate bringing the cluster to its knees, the timeout period will allow the connector to recover rather then be stuck waiting.
For the cluster-health request - 10 seconds should be enough.
For the Search request - 30 minutes sounds right (would be the same as the default timeout for Presto queries).

Reply:
Given than lambda limits runtime to 15 minutes currently, we should probably set our timeout to be  less than 15 minutes (maybe 12 by default per-shard)